tests/governance_js.py

```diff
@@ -345,7 +345,7 @@ def test_actions(network, args):
         r = c.post("/gov/proposals.js", valid_set_member_data)
         assert r.status_code == 200, r.body.text()
 
-        valid_rekey_ledger = proposal(action("rekey_ledger"))
+        valid_rekey_ledger = proposal(action("trigger_ledger_rekey"))
```

Comment:
Minor drive-by rename to match what we specc'ed in https://github.com/microsoft/CCF/discussions/2169#discussioncomment-373490